src/types/excelreader.rs

```diff
@@ -38,14 +38,13 @@ impl ExcelReader {
         format!("ExcelReader<{}>", &self.path)
     }
 
-    #[args(
+    #[pyo3(signature = (
         name,
-        "*",
```

Comment:
Not sure about this "*". Maybe you have a idea @lukapeschke

Reply:
Please leave it, it's the python syntax for named-only arguments

Reply:
done